roles/hosts/tasks/type-block.yml

```diff
@@ -2,7 +2,7 @@
 - name: Get /etc/hosts file
   ansible.builtin.slurp:
     src: "{{ hosts_file }}"
-  register: hosts
+  register: hosts_entries
```

Comment:
you'll also have to change the references to this variable below. I'm surprised that the test is still passing, seems like the coverage could be improved

Reply:
Something went wrong with my push force. Now it should be fixed.